plugins/cost-insights/src/components/AlertInsights/AlertInsightsHeader.tsx

```diff
@@ -30,10 +30,11 @@ export const AlertInsightsHeader = ({
   subtitle,
 }: AlertInsightsHeaderProps) => {
   const classes = useStyles();
-  const { ScrollAnchor } = useScroll(DefaultNavigation.AlertInsightsHeader);
+  const [, , ScrollAnchor] = useScroll();
```

Comment:
Feels slightly weird, maybe `useScroll` should return an object for easy destructuring only what we need?

Reply:
Since the hook now returns state and a setter - instead of a callback and component - array destructuring felt more idiomatic. 

The hook really just provides a component as a convenience. This could be moved into a util?